package/yast2-storage-ng.spec

```diff
@@ -28,6 +28,8 @@ Requires:	yast2 >= 4.0.61
 Requires:	yast2-ruby-bindings >= 4.0.6
 # function light_probe
 Requires:	libstorage-ng-ruby >= 3.3.254
+# communicate with udisks
+Requires:	rubygem(dbus)
```

Comment:
Is it `rubygem(ruby-dbus)` ? And I think the `rubygem(%rb_default_ruby_abi:ruby-dbus)`  way should be preferred as that will enforce the system Ruby version. (Currently there is only a single Ruby version in SLE15/Leap15 but this will be future proof if Tumbleweed adds a new experimental Ruby version.)

Reply:
No idea about this Ruby packing stuff. I just copied it from the BuildRequires.

Reply:
The name of the gem is "ruby-dbus", the spec file needs to be fixed.